doc/cli.rst

```diff
@@ -244,6 +244,7 @@ Trigger a workflow on deployment <DeploymentId>.
      yorc deployments workflows execute <DeploymentId> [flags]
```

Comment:
an example could be useful for executing custom workflows as for custom commands to see the required json format